articles/protocols/saml/adfs.md

```diff
@@ -4,202 +4,137 @@ toc: true
 topics:
   - saml
   - adfs
+  - active-directory-federation-services
 contentType:
   - how-to
 useCase:
   - add-idp
+  - saml-adfs
 ---
 
-# ADFS SAML Connection
+# Setup a Microsoft ADFS SAML Connection
 
-Auth0 allows you to create a custom <dfn data-key="security-assertion-markup-language">SAML</dfn> connection to Microsoft's Active Directory Federation Services. In addition to getting a bit more flexibility when configuring your mappings, the SAML Connection allows you identity provider-initiated flows (this is something that you cannot do with WS-Fed).
+Create a custom <dfn data-key="security-assertion-markup-language">SAML</dfn> connection to Microsoft's Active Directory Federation Services (ADFS) to get more flexibility when configuring your mappings. The SAML Connection allows you identity provider-initiated flows which you cannot do with WS-Fed.
```

Comment:
I would change "The SAML Connection allows you identity provider-initiated flows which you cannot do with WS-Fed." to "In addition, SAML connections allow [IdP-Initiated flows](protocols/saml/idp-initiated-sso), which are not supported with regular ADFS-type connections.". Or, considering that we don't recommend IdP-initiated flows, I'd just remove that sentence altogether.

Reply:
Removed